spec/features/user_can_sign_in_spec.rb

```diff
@@ -7,28 +7,30 @@
   end
 
   scenario "A user can sign in with valid credentials" do
-    visit "/"
+    visit "/session/new"
     fill_in :email, with: "test_email@keepingitrails.com"
     fill_in :password, with: "pa55w0rd"
     click_button "Log In"
     expect(page).to have_content("Welcome test_first_name")
   end
 
-  scenario "A user signs in with invalid email address" do
+  #not implementing the below feature in this commit:
 
-    visit "/"
-    fill_in :email, with: "test_email_other@gmail.com"
-    fill_in :password, with: "pa55w0rd"
-    click_button "Log In"
-    expect(page).to raise_error("Sorry, we do not recognise this email address")
-  end
-
-  scenario "A user signs in with incorrect password" do
-
-    visit "/"
-    fill_in :email, with: "test_email@keepingitrails.com"
-    fill_in :password, with: "incorrect password"
-    click_button "Log In"
-    expect(page).to raise_error("Sorry, please check your password and try again")
-  end
+  # scenario "A user signs in with invalid email address" do
+  #
+  #   visit "/session/new"
+  #   fill_in :email, with: "test_email_other@gmail.com"
+  #   fill_in :password, with: "pa55w0rd"
+  #   click_button "Log In"
+  #   expect(page).to have_content("Sorry, we do not recognise this email address")
+  # end
+  #
+  # scenario "A user signs in with incorrect password" do
+  #
+  #   visit "/session/new"
+  #   fill_in :email, with: "test_email@keepingitrails.com"
```

Comment:
you knew it was coming...... can we extract this to a web helper with very clear scenario names? we should have one web helper method for each incorrect user input (eg. email address already exists, password too long etc etc)